recipes/base_install.rb

```diff
@@ -235,3 +235,6 @@
 
 # Install FSx options
 include_recipe "aws-parallelcluster::_lustre_install"
+
+# Install EFA
+include_recipe "aws-parallelcluster::_efa_install"
```

Comment:
just for consistency I would move the OS check here like we do for the fsx recipe

Reply:
I don't understand what you mean? FSx recipe checks the os in the `base_config.rb`, not in `base_install.rb`. We install efa drivers in `base_install.rb` so it'll be built into the ami, or installed at runtime for a custom ami.

Reply:
yeah, I meant to move the check
`if (node['platform'] == 'centos' && node['platform_version'].to_i >= 7) || node['platform'] == 'amazon'`
outside the recipe _efa_install, so it's not even called

`if (node['platform'] == 'centos' && node['platform_version'].to_i >= 7) || node['platform'] == 'amazon'
  include_recipe "aws-parallelcluster::_efa_install"
end
`